### ISSUE TITLE
Bottom-line overlap when generating cards on linux

### ISSUE THREAD

**Issue by Gikkman** [Haven-King/Proximity#30]
When generating a card of of the Ubuntu docker image, I get the following bottom line:

![image](https://user-images.githubusercontent.com/3306495/135919657-6b4beeb3-13ab-4e85-933f-e8a236311d34.png)

But when I generate images from my Mac, I get the following bottom line:

![image](https://user-images.githubusercontent.com/3306495/135919704-4d093861-4370-4385-9bc2-b69075b3fd9a.png)

I don't know if this is an issue with my docker image, or if it is an image with native fonts on Ubuntu

**Comment by fklemme** [comment]
I think I figured it out. Microsoft's _Calibri_ font is not installed by default on Ubuntu. There is a replacement font instead that seems to interpret the font size differently. I found a website explaining how to grab the font from Microsoft and install it on Linux [[1](https://kodejava.org/how-do-i-install-calibri-font-in-ubuntu/)]. I tested it and it runs just fine afterwards.

Is this the best solution? I'm not sure. I would probably rather consider to use a different font in the template, maybe something that is free and installed on most OSs by default. In the end, it's just a small custom line where no special font is needed to mimic the look of a Magic card.
